### DISCUSSION TITLE
Ability to fill/empty cauldrons with dispensers

### DISCUSSION THREAD

**Discussion by RhythmicSys** [PurpurMC/Purpur#585] [__Dismissed__] [answered]
The ability to fill cauldrons with water/lava/powdered snow, or to empty them with dispensers, would make for some really nice autofarms

**Comment by Keeg-Turner** [comment, marked as answer]
This is in [PurpurExtras](https://github.com/PurpurMC/PurpurExtras) now, though they seem to have forgotten to mention the config option in the wiki.

**Reply by RhythmicSys** [reply]
I made this one as well XD but thank you for lining the plugin, I meant to do that a long while ago but forgot
